Improve markdown files

raballew opened the pull request:
The following changes have been applied:

- Use a maximum line length of 120 chars
- Fix several typos
- Improve readability of doc/test/hirte_e2e.md
- Improve readability of project description
- Remove stale file doc/test/orch_test_notes
- Use same spelling for Hirte

Yarboa reviewed (approved):
LGTM

mwperina commented on README.developer.md:
```diff
@@ -21,10 +21,10 @@ sudo dnf config-manager --set-enabled crb
-[installation guide](https://github.com/DavidAnson/markdownlint-cli2#install) and use the most appropriate way of
-installation for your setup.
+[markdownlint-cli2](https://github.com/DavidAnson/markdownlint-cli2) can be used for static analysis of markdown files.
+Unfortunately it's not available as RPM, so please check its [installation
```
Personally I'd leave URL links on a single line rather than having new line inside

mwperina commented on README.md:
```diff
@@ -16,18 +17,18 @@ to generate systemd service configuration to run a container.
-If you are not familiar with the development process you can read about it in
-[Get started with GitHub](https://docs.github.com/en/get-started).
+If you are not familiar with the development process you can read about it in [Get started with
+GitHub](https://docs.github.com/en/get-started).
```
Personally I'd leave URL links on a single line rather than having new line inside

mwperina commented on README.md:
```diff
@@ -16,18 +17,18 @@ to generate systemd service configuration to run a container.
-To submit a bug or suggest an enhancement for hirte please use
-[GitHub issues](https://github.com/containers/hirte/issues).
+To submit a bug or suggest an enhancement for Hirte please use [GitHub
+issues](https://github.com/containers/hirte/issues).
```
Personally I'd leave URL links on a single line rather than having new line inside

mwperina commented on README.md:
```diff
@@ -16,18 +17,18 @@ to generate systemd service configuration to run a container.
-If you have any other questions, please join
-[CentOS Automotive SIG mailing list](https://lists.centos.org/mailman/listinfo/centos-automotive-sig/) and ask there.
+If you have any other questions, please join [CentOS Automotive SIG mailing
+list](https://lists.centos.org/mailman/listinfo/centos-automotive-sig/) and ask there.
```
Personally I'd leave URL links on a single line rather than having new line inside

mwperina reviewed (commented):
Other than splitting URL over multiple lines it looks good to me

raballew committed:
Improve markdown files

The following changes have been applied:
- Use a maximum line length of 120 chars
- Fix several typos
- Improve readability of doc/test/hirte_e2e.md
- Improve readability of project description
- Remove stale file doc/test/orch_test_notes
- Use same spelling for Hirte

Signed-off-by: Paul Wallrabe <54737071+raballew@users.noreply.github.com>

mwperina reviewed (approved):
+1
